OtripleS.Web.Api.Tests.Unit/Services/Calendars/CalendarServiceTests.Exceptions.Delete.cs

```diff
@@ -0,0 +1,164 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Data.SqlClient;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+using OtripleS.Web.Api.Models.Calendars;
+using OtripleS.Web.Api.Models.Calendars.Exceptions;
+using Xunit;
+
+namespace OtripleS.Web.Api.Tests.Unit.Services.Calendars
+{
+    public partial class CalendarServiceTests
+	{
+		[Fact]
+		public async Task ShouldThrowDependencyExceptionOnDeleteWhenSqlExceptionOccursAndLogItAsync()
+		{
+			// given
+			Guid randomCalendarId = Guid.NewGuid();
+			Guid inputCalendarId = randomCalendarId;
```

Comment:
These two variables should just be called as one `someCalendarId = Guid.NewGuid()`
We call these *someEntity* because they have no real significance to what we are trying to test.